src/editline.c

```diff
@@ -1043,16 +1067,17 @@ static el_status_t meta(void)
             char seq[4] = { 0 };
             seq[0] = tty_get();
 
+            /* \e[1~ */
             if (seq[0] == '~')
                 return beg_line(); /* Home */
 
             for (c = 1; c < 3; c++)
                 seq[c] = tty_get();
 
             if (!strncmp(seq, ";5C", 3))
-                return fd_word(); /* Ctrl+Right */
+                return fd_word(); /* \e[1;5C = Ctrl+Right */
             if (!strncmp(seq, ";5D", 3))
-                return bk_word(); /* Ctrl+Left */
+                return bk_word(); /* \e[1;5d = Ctrl+Left */
```

Comment:
inconsistent casing in this comment vs the line above. please check which one is correct :)